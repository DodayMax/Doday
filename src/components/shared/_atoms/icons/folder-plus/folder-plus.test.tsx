import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@components';

describe('FolderPlus', () => {
  it('renders correctly without props', () => {
    const folderPlus = shallow(<Icons.FolderPlus />);

    expect(folderPlus).toMatchSnapshot();
  });
});