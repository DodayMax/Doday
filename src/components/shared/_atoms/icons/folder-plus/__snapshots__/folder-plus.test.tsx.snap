// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FolderPlus renders correctly without props 1`] = `ShallowWrapper {}`;
